src/platform/windows/window.rs

```diff
@@ -239,6 +239,33 @@ impl Window {
             }
         }
     }
+    
+    /// See the docs in the crate root file.
```

Comment:
You don't need to add this comment, since #548 completely removes them... they don't seem to be at all helpful, especially since these are doc comments on private methods.

More importantly, what happens when you use this method while the window is fullscreen?